### PR TITLE
fix: going to do some testing to fix dthis directly from the github action

### PR DESCRIPTION
**Pull request by Sanix-Darker**
## WHAT

- Some fixing

### CODE REVIEW

**Comment by Sanix-Darker** [comment]
Now we need to put a valid TOKEN for the appropriate chat !

**Comment by Sanix-Darker** [comment]
F*****************CCCCCCCCCCCCCCKKKKKKK

File: .github/workflows/report_jobs.yaml [hunk 1]
```diff
@@ -17,12 +16,13 @@ jobs:
...
         run: |
-          ./report_jobs.sh > report_jobs
+          ./report_jobs.sh > raw
+          cat ./raw
 
       - name: send jobs report to osscameroon group
         uses: appleboy/telegram-action@v0.1.1
         with:
-          to: ${{ secrets.TELEGRAM_REMOTEJOBS_GROUP_ID }}
+          to: ${{ secrets.TELEGRAM_OSSCAMEROON_GROUP_ID }}
```
Comment (elhmn) [inline .github/workflows/report_jobs.yaml:25]: Should we send them to the oss channel instead ?

File: .github/workflows/report_jobs.yaml [hunk 2]
```diff
@@ -1,8 +1,7 @@
+---
 
 name: telegram jobs report
-on:
-  schedule:
-    - cron: "0 15 * * *"
+on: [push]
```
Comment (elhmn) [inline .github/workflows/report_jobs.yaml:4]: This should run on a schedule

File: .github/workflows/report_jobs/report_jobs.sh [hunk 3]
```diff
@@ -5,59 +5,49 @@ set -e
...
+        url: .url,
+        tags: .tags,
+        title: .title,
+        company_name: .company_name
+    }'
+    # | select(.tags[] | (
+    #     select(. | contains("IT")) or
+    #     select(. | contains("Software")) or
+    #     select(. | contains("Web Development"))
+    # ))'
```
Comment (elhmn) [inline .github/workflows/report_jobs/report_jobs.sh:21]: Why do we keep this comments ?

File: .github/workflows/report_jobs.yaml (hunk 2 above)
Comment (Sanix-Darker) [reply .github/workflows/report_jobs.yaml:4]: For testing purposes

**Comment by Sanix-Darker** [comment]
Closing this in favor of :https://github.com/osscameroon/global-github-actions/pull/111

**Pull request closed without merging**